### PR TITLE
Dockerization

### PR DESCRIPTION
**Pull request by mkorcy**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -102,7 +98,7 @@ jobs:
     - name: Setup hyrax test environment
       run: |
         bundle exec rake hydra:test_server &
-        sleep 150
+        sleep 180
```
Comment (mkorcy) [inline .github/workflows/ci.yml:101]: I find the GHA container is a little slower so I bumped the sleep.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+RUN apt-get update && apt-get install -y wget gnupg
+RUN apt-get install -y wget apt-transport-https gnupg
+RUN echo "deb https://packages.adoptium.net/artifactory/deb $(awk -F= '/^VERSION_CODENAME/{print$2}' /etc/os-release) main" | tee /etc/apt/sources.list.d/adoptium.list
+
+RUN wget -qO - https://packages.adoptium.net/artifactory/api/gpg/key/public| apt-key add - \
+    && apt-get update \
+    && apt-get install -y temurin-8-jdk
+
+RUN update-java-alternatives --set /usr/lib/jvm/temurin-8-jdk-amd64
```
Comment (mkorcy) [inline Dockerfile]: I suspect we may start needing this change in other places, worth noting: https://adoptium.net/blog/2023/07/adoptopenjdk-jfrog-io-has-been-deprecated/

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+# Create a symbolic link to the Chrome binary
+RUN ln -s /usr/local/bin/thorium /usr/local/bin/google-chrome
+RUN ln -s /usr/local/bin/thorium /usr/local/bin/chrome
+RUN ln -s /usr/local/bin/thorium /usr/local/bin/chromium
+
+# Cleanup unnecessary files
+RUN apt-get remove -y wget unzip && apt-get clean
+
+# Verify the installation
+RUN google-chrome --version
```
Comment (rawOrlando) [inline Dockerfile]: We can get rid of this now.

**Comment by mkorcy** [comment]
@rawOrlando -- ok can you try this again ?

File: docs/future_notes.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+```
+## Download and extract Google Chrome
+
+attempts using thorium
+
+```
+RUN wget https://github.com/Alex313031/thorium/releases/download/M114.0.5735.134/thorium-browser_114.0.5735.134_amd64.zip && \
+    unzip chrome-linux.zip -d /usr/local/bin/ && \
+    rm chrome-linux.zip
+```
```
Comment (rawOrlando) [inline docs/future_notes.md:23]: I am getting an error. I can't resolve from this. 
``` 
failed to solve: executor failed running [/bin/sh -c wget https://github.com/Alex313031/thorium/releases/download/M114.0.5735.134/thorium-browser_114.0.5735.134_amd64.zip      unzip chrome-linux.zip -d /usr/local/bin/]: exit code: 4 
```

Comment (mkorcy) [reply docs/future_notes.md:23]: Huh? That change isn't in a file, its just in markdown

Comment (mkorcy) [reply docs/future_notes.md:23]: Did you pull updates?

Comment (mkorcy) [reply docs/future_notes.md:23]: @rawOrlando i don't understand where this got left off, can you take another pass through this, and tell me what the issue is? The thorium thing is in a markdown file of notes.

**Comment by rawOrlando** [comment]
I got this to work locally with two small changes. 

I needed to install rubygems-update before `gem update --system`
`RUN gem install rubygems-update -v 3.4.22`

I needed to remove `therubyracer` gem form the gem file.

**Pull request merged**